README.md

```diff
@@ -734,45 +723,15 @@ which will load the `foo.csv` file as a dataframe.
 
 ##### Step 4.3: Loading from `Postgres`
 
-In contrast to `S3` resources, `postgres` resources need additional options to be
-defined for their loading.
-
-Specifically, you need to define a data model, defining the table, the columns and
-their respective SQL types. This is necessary as a different reader is utilised in
-the case of postgres (this need will be addressed in future releases).
+Likewise to `S3` resources, `postgres` resources need the same number of options to be defined for their loading.
 
-The data model definition is also defined in `io.py` and looks like:
-
-```python
-"""
-A module for defining sql_alchemy models.
-"""
-__all__ = ["BarDataModel"]
-
-from sqlalchemy import Column, Float, String
-from sqlalchemy.ext.declarative import declarative_base
-
-Base = declarative_base()
-
-
-class BarDataModel(Base):
-    """
-    Sql_alchemy model for Bar table
-    """
-
-    __tablename__ = "bar"
-
-    column_a = Column(String(64), primary_key=True)
-    column_b = Column(String(64))
-    column_c = Column(Float)
-    column_d = Column(Float)
-
-```
+**Implicitly, dynamicio is able to infer data model from the schema yml files of the source key provided rather than requiring that the schema is explicitly defined.**
+This data model defines the table, the columns andtheir respective SQL types.
```

Comment:
Great update to the dos.

`nit`: small typo "andtheir" --> "and their"

Reply:
yeah sorry my bad